### ISSUE TITLE
Delete button should actually delete highlights

### ISSUE THREAD

**Issue by Stvad** [DanilaFe/matrix-highlight#30]
I've noticed that "delete button" is currently sending an edit to "hide" a highlight, but doesn't actually delete it 🤔.  
This seems misleading, and it should either actually delete the event (with user confirmation) or UI should communicate that "hiding" is what's happening (but then people would still plausibly want to delete highlights)

**Comment by DanilaFe** [comment]
Originally I believe I was redacting events; redactions, however, make things like "undo" impossible because they physically delete an event from the homeserver. I thus definitely think that "hiding" is what should happen, but maybe the trash icon should be replaced with something else (crossed out eye?).

**Comment by Stvad** [comment]
Ah, I see. Yeah, a different icon would definitely help. I think there is also a place for actual deletion/reduction of the events.   
Also, as far as I can tell there is no way to "unhide" the highlight rn?

**Comment by DanilaFe** [comment]
Yes, there's no way to unhide highlights -- I think I would need some sort of "undo stack" to make that work, but that seems pretty involved (or maybe you can browse hidden highlights in the menu?). I'll change the icon to be something different, though.

**Comment by DanilaFe** [comment]
I am using the "crossed out eye" icon as of 945f4d8b3fc66524aaa16edfbd34a54d612dd0d9. As for properly deleting highlights: I think this will need to come as part of a more proper "admin user interface".
